packages/salesforcedx-vscode-core/src/commands/forceSourceDeploySourcePath.ts

```diff
@@ -128,8 +128,7 @@ export class LibraryDeploySourcePathExecutor extends DeployRetrieveLibraryExecut
       )) as string;
       const registryAccess = new RegistryAccess();
       const components = registryAccess.getComponentsFromPath(response.data);
-      const deployPromise = this.sourceClient.tooling.deploy({
-        components,
+      const deployPromise = this.sourceClient.tooling.deploy(components, {
```

Comment:
Should this be changed? I don't think we're using the new version of the library yet.